appserver/packager/glassfish-ejb/pom.xml

```diff
@@ -103,6 +103,10 @@
             <artifactId>ejb-http-admin</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-el-api</artifactId>
+        </dependency>
```

Comment:
Not sure this should be in here.
Would appserver-core, glassfish-common, or glassfish-jcdi not be a better place?

Reply:
yep I moved to the appserver-core, and worked correctly, thanks